samples/apps/copilot-chat-app/webapp/src/redux/features/users/UsersState.ts

```diff
@@ -0,0 +1,17 @@
+// Copyright (c) Microsoft. All rights reserved.
+
+export interface UsersState {
+    users: Users;
+}
+
+export const initialState: UsersState = {
+    users: {},
+};
+
+export type Users = Record<string, UserData>;
```

Comment:
building on my previous comment, this seems it should be like below to represent the fact that the data my not be there.
```suggestion
export type Users = Record<string, UserData | undefined>;
```